src/test/java/org/mule/extension/db/integration/connectivity/negative/SqlServerNegativeConnectivityTestCase.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.db.integration.connectivity.negative;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.mule.extension.db.api.exception.connection.DbError.CANNOT_REACH;
+import static org.mule.extension.db.api.exception.connection.DbError.INVALID_CREDENTIALS;
+import static org.mule.extension.db.api.exception.connection.DbError.INVALID_DATABASE;
+import static org.mule.tck.junit4.matcher.ErrorTypeMatcher.errorType;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+@Ignore
```

Comment:
I'll merge it but need to un-ignore this some time soon

Reply:
This is because we can't run SQL Server tests by default :/

Reply:
there's a central configuration on the test framework in which we enable the tests to run